OpenSim/Common/XMLDocument.cpp

```diff
@@ -54,7 +54,8 @@ using namespace std;
 // 30502 for Changing serialization of Geometry
 // 30503 for Changing serialization of Ground
 // 30505 for Changing serialization of Joint to create offset frames
-const int XMLDocument::LatestVersion = 30505;   
+// 30506 for Changing serialization of PrescribedForce to use Connectors
+const int XMLDocument::LatestVersion = 30506;   
```

Comment:
I am not sure I follow. Is the issue that if I already have a 30505 update and I now want to have an additional update then their is ambiguity? I would sort of agree, but then the argument to increment should be that there is an existing update for the current version and you want to keep the next update separate.

In this case `PrescribedForce` had no versions to update at all, so why not include the update in 30505? I have not been incrementing the version numbers for updates I have been making and treating 30505 as the moving format as we make changes. So I have not been consistent with the a rule to increment the version for every update change. I think that would be a good rule to adopt going forward but maybe not during the overhaul itself.

Thanks for discussing in person @aymanhab and @chrisdembia. For now, I think we agreed to increment when updates to be applied to a given class already have an update for the latest version, then it makes sense to keep new changes separate and bump up the version number. 

We also agreed we should bump the version number up prior to having a version we are willing to support (e.g. for the Workshop).